zeq/event.cpp

```diff
@@ -59,4 +54,14 @@ const flatbuffers::Parser& Event::getParser() const
     return _impl->parser;
 }
 
+void Event::setData( const void* data, const size_t size )
+{
+    _impl->setData( data, size );
+}
+
+void Event::setData( const ConstByteArray& data, const size_t size )
```

Comment:
The caller is the library, no user code should call that. So one version is for serialize, the other for deserialize?